nipype/interfaces/io.py

```diff
@@ -212,12 +212,12 @@ class DataSinkInputSpec(DynamicTraitedSpec, BaseInterfaceInputSpec):
     """
 
     # Init inputspec data attributes
-    base_directory = Directory(desc="Path to the base directory for storing data.")
+    base_directory = Str(desc="Path to the base directory for storing data.")
     container = Str(desc="Folder within base directory in which to store output")
     parameterization = traits.Bool(
         True, usedefault=True, desc="store output in parametrized structure"
     )
-    strip_dir = Directory(desc="path to strip out of filename")
+    strip_dir = Str(desc="path to strip out of filename")
```

Comment:
What is the effect of this change? Was something failing?

Reply:
No, I don't think this will make any difference in proper use cases.

But since we are setting `base_directory` trait to `Str` to be faithful to the user's inputs, it would not make sense to disable input processing on one path input but not the other.